sph/__init__.py

```diff
@@ -0,0 +1,5 @@
+#pylint: disable=missing-module-docstring
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
let's move this section to the demo scripts.
Log config should be made there and not on the src lib.